packages/vx-gradient/src/gradients/GradientDarkgreenGreen.tsx

```diff
@@ -4,7 +4,7 @@ import LinearGradient, { LinearGradientProps } from './LinearGradient';
 /**
  * All props pass through to `<LinearGradient {...props} />`
  */
-export default function GradientDarkGreen({
```

Comment:
😄

Reply:
docgen barfs (not really, it just doesn't parse props correctly) if the component name + file name don't match 😅